pdh_check/datadog_checks/pdh_check/data/conf.yaml.example

```diff
@@ -2,32 +2,28 @@ init_config:
 
 instances:
 
+## The following example fetches the number of processes and users:
```

Comment:
This lies inside the instances part and should have the same as the rest of the content. (2 spaces)